src/Passes/docs/src/DeveloperGuide/WritingRuleTests.md

```diff
@@ -0,0 +1,120 @@
+# Writing rule tests
+
+To make it easy to write tests for transformation rules, we have created two components to easy the burden of writing tests: `RuleSetProfile` and `IrManipulationTestHelper`. The `RuleSetProfile` is a profile that is dynamically defined when instatiated through a configuration lambda function.
+
```

Comment:
1. I fail to find `RuleSetProfile` and `IrManipulationTestHelper`.
2.
```suggestion
To make it easy to write tests for transformation rules, we have created two components to ease the burden of writing tests: `RuleSetProfile` and `IrManipulationTestHelper`. The `RuleSetProfile` is a profile that is dynamically defined when instatiated through a configuration lambda function.

```

Reply:
`RuleSetProfile` was previously refactor to `DefaultProfileGenerator`. The documents have been updated to reflect this.